### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by NileshPant1999**
## Description
Added documentation for test files

If your pull request closes a GitHub issue, then set its number below.

## Type of change

Please mark options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist:

* [x] I did follow the [contribution guidelines](https://github.com/OpenMined/PySyft/blob/master/CONTRIBUTING.md)
* [x] I have commented my code following [Google style](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html).
* [ ] I have added tests for my changes

### CODE REVIEW

**Comment by AlanAboudib** [comment]
Any updates on this @NileshPant1999

**Comment by NileshPant1999** [comment]
> Any updates on this @NileshPant1999

I am woking on it. will do it by tommorow

**Comment by Nilanshrajput** [comment]
@NileshPant1999  are you still working on this?

File: tests/test_notebooks.py [hunk 1]
```diff
@@ -4,9 +4,9 @@
 import papermill as pm
 
 
+"""selecting all the notebook in the tutorial folder."""
 all_notebooks = [n for n in glob.glob("tutorials/*.ipynb", recursive=True)]
```
Comment (AlanAboudib) [inline tests/test_notebooks.py:9]:
```suggestion
# Selecting all the notebook in the tutorial folder.
all_notebooks = [n for n in glob.glob("tutorials/*.ipynb", recursive=True)]

```

**Comment by AlanAboudib** [comment]
I will close this PR since it is stale. @NileshPant1999 your contribution will be welcome at any time :)

**Pull request closed without merging**